src/test/shared/uploadToAwsCommand.test.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { createWizardTester, WizardTester } from './wizards/wizardTestUtils'
+import { UploadToAwsWizard, UploadToAwsWizardState } from '../../shared/uploadToAwsCommand'
+
+describe('UploadToAwsWizard', function () {
+    let tester: WizardTester<UploadToAwsWizardState>
+
+    beforeEach(function () {
+        tester = createWizardTester(new UploadToAwsWizard())
+    })
+
+    it('shows region prompt when S3 is selected', function () {
```

Comment:
the region menu is not really the main point of this wizard, so these test names are confusing